subprojects/groovy-groovydoc/src/test/groovy/org/codehaus/groovy/tools/groovydoc/testfiles/JavaStaticNestedClassWithDiamond.java

```diff
@@ -18,12 +18,17 @@
  */
 package org.codehaus.groovy.tools.groovydoc.testfiles;
 
-public abstract class JavaStaticNestedClassWithDiamond<E> {
+public class JavaStaticNestedClassWithDiamond<E> {
```

Comment:
Why should `abstract` be removed?

Reply:
It did not see any reason for the test class being abstract. Please let me know if I should revert the line in question.

Reply:
OK. I see. It does not matter IMO.